srcpkgs/google-chrome/template

```diff
@@ -48,3 +49,8 @@ do_install() {
 	rm -rf ${DESTDIR}/etc
 	rm -rf ${DESTDIR}/opt/google/chrome/cron
 }
+
+post_install() {
+	curl ${_licenseUrl} | html2text >> ${DESTDIR}/google-chrome-eula.md
```

Comment:
This leaves a google-chrome-eula.md file at `/` which is not acceptable.  Please curl this to your local directory and then vinstall it from there.